source/src/python/PyRosetta/src/test/T011_SaveIO.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+# :noTabs=true:
+# -*- coding: utf-8 -*-
+
+# (c) Copyright Rosetta Commons Member Institutions.
+# (c) This file is part of the Rosetta software suite and is made available under license.
+# (c) The Rosetta software is developed by the contributing members of the Rosetta Commons.
+# (c) For more information, see http://www.rosettacommons.org. Questions about this can be
+# (c) addressed to University of Washington CoMotion, email: license@uw.edu.
+
+## @file   T011_SavePDB.py
+## @brief  Tests for the saving functionality of PyRosetta
+## @author Rocco Moretti
+
+from __future__ import print_function
+
+import os
+import pyrosetta
+import pyrosetta.rosetta as rosetta
+import tempfile
+import unittest
+import json
+
+try:
+    import lzma as xz
+    _skip_xz = False
+except ImportError:
+    _skip_xz = True
+
+
```

Comment:
Could you please add the standard lines we include in all PyRosetta tests?
They ensure that no unexpected output accumulates and that runs are deterministic.
(I know the test code already calls init(), but in this case uniformity is probably preferable to purity.)

```python
init(extra_options = "-constant_seed")  # WARNING: option '-constant_seed' is for testing only! MAKE SURE TO REMOVE IT IN PRODUCTION RUNS!!!!!
import os; os.chdir('.test.output')
```
Thanks,